services/spar/package.yaml

```diff
@@ -131,6 +131,7 @@ executables:
     dependencies:
       - aeson-qq
       - async
+      - brig
```

Comment:
hm, this is not nice to depend on another service entirely. Could you extract whichever functions and types you need into `types-common`, or `brig-types`, or `wire-api`, then remove this depdency?

Reply:
I removed the dependency (see comment on Makefile)